### ISSUE TITLE
uBlock Origin filters not active upon starting Chrome to open hyperlink

### ISSUE THREAD

**Issue by zfrm** [gorhill/uBlock#1913]
### Describe the issue

If Google Chrome isn't running and a hyperlink is clicked (in a document/irc/wherever) in order to open it in the default browser, UBO doesn't catch it upon opening it for the first time; only after refreshing the page, after loading, does UBO apply its appropriate filters.

This happens with filters in the "3rd-party filters" lists (including malware domains) and "My filters".
### One or more specific URLs where the issue occurs

Picking any entry from a filter list, closing the browser and opening selected the URL as a hyperlink from another program appears to replicate the behavior every time.

Tested it by adding "google.com" and "google.fi" to "My filters" and clicking a "http://www.google.com" hyperlink.
### Steps for anyone to reproduce the issue
1. Pick an URL to block from a 3rd-party filter list OR create your own (example.com)
2. Check that the filter works (prevents loading the page) by typing the URL in Chrome's address bar
3. Close the browser
4. Add the URL as a hyperlink into a document in the program of your choice
5. Click the hyperlink so that it opens in your default browser (which should be Chrome in this instance)
6. Provided that the issue is replicated and the selected site isn't blocked, refresh the page - it should be now caught by UBO
### Your settings
- Browser/version: Google Chrome Version 52.0.2743.116 m (64-bit)
- uBlock Origin version: 1.8.4
- OS: Windows 7 Pro x64 SP1
##### Your filter lists

All 3rd-party filter lists + FIN, minus "uBlock filters Experimental" and "Easylist without element hiding rules"
##### Your custom filters (if any)

For the aforementioned testing purposes:
google.com
google.fi

**Comment by gorhill** [comment]
Browser limitation, the browser does not wait for extensions to be "ready" before opening tabs.

**Comment by gorhill** [comment]
Duplicate of #1327.
